src/Yaapii.Xml.Xambly/Directive/XpathDirective.cs

```diff
@@ -88,7 +94,28 @@ public ICursor Exec(XmlNode dom, ICursor cursor, IStack stack)
             //{
             //    targets = this.Traditional(query, dom, cursor);
             //}
-            targets = this.Traditional(query, dom, cursor);
+
+            // CSA: This fork is only needed if the cursor points to an XmlNode which is not existing in the dom. 
```

Comment:
@aveXcaesar I think infos like you wrote here should go into a comment section of the issue or as comment in the PR.
These comments look like personal notes on the learnings you made while working on this issue. 
Of course this are important values for everybody but i think the place were they are document is wrong.
I will remove them from the code.

Reply:
Next time I have to think about it again. This is a very complicated section and it takes much time to understand it.
You are right to clean up the code. But in this class I think we should keep it...

Reply:
@avexcaesar this is a public repo, ypur personal username must not appear here!